resources/js/Pages/Dashboard.vue

```diff
@@ -27,6 +28,7 @@ const props = defineProps({
                         <div class="grid grid-cols-3 gap-4">
                             <SelectedRunner :company="company" class="col-span-3 " />
                             <Orders :delivery-moment="deliveryMoment" class="col-span-3"></Orders>
+                            <done-orders></done-orders>
```

Comment:
camelcase of pascal kiezen. Niet beide (moet trouwens global verwerkt worden dus bespreken we volgende week)